proposals/drafts/elections.rst

```diff
@@ -43,46 +255,502 @@ extras
+
+contest_id
+    References the ``BallotMeasureContest``, ``CandidateContest``, ``PartyContest`` or ``RetentionContest`` in which the ballot selection is an option.
+
+sequence_order
+    **optional**
+    The order in which a selection can be listed on the ballot or in results of the associated contest (integer).
+
+vote_counts
+    **optional**
+    **repeated**
+    Count of votes for the ballot selection, broken down by type. Has the following properties:
+
+    count
+        Vote count (integer)
+
+    type
+        Enumerated among:
+
+        * absentee: For any/all types of absentee, generally when absentee is not broken out into specific types.
+        * absentee-fwab: A type of absentee; for Federal Write-in Absentee Ballots.
+        * absentee-in-person: A type of absentee; for absentee ballots cast in-person, e.g., at a county office.
+        * absentee-mail: A type of absentee; for postal mail absentee ballots.
+        * early: For ballots cast during early voting periods.
+        * election-day: For ballots cast on election day.
+        * provisional For challenged ballots.
+        * total: For the total of all ballot types.
+        * uocava: For ballots from UOCAVA (Uniform and Overseas Citizens Assistance in Voting Act) voters.
+        * write-in: For write-ins on ballots.
+        * other: Used when the type of counting item is not included in this enumeration.
+
+    other_type
+        Allows for cataloging a new type of vote count type option, when type is specified as "other" (string).
+
+
+BallotMeasureSelection
+----------------------
+
+A subclass of ``BallotSelectionBase`` representing a ballot option that a voter could select in a ballot measure contest.
+
+id
+    Open Civic Data-style id in the format ``ocd-ballotmeasureselection/{{uuid}}``.
+
+selection
+    Selection text for the option on the ballot , e.g., "Yes", "No", "Recall", "Don't recall" (string).
+
+
+CandidateSelection
+------------------
+
+A subclass of ``BallotSelectionBase`` representing an option on the ballot that a voter could select in a candidate contest, e.g., a particular candidate or "ticket".
+
+id
+    Open Civic Data-style id in the format ``ocd-candidateselection/{{uuid}}``.
+
+candidates
+    **repeated**
+    Lists each ``Candidate`` associated with the ballot selection. The number of candidates is unbounded in cases where the ballot selection is for a ticket, e.g. "President/Vice President", "Governor/Lt Governor". Has the following properties:
+
+        candidate_id
+            References the ``Candidate``.
+
+        post_id
+            References the ``Post`` that represents the public office for which the candidate is competing.
+
+endorsement_party_ids
+    **optional**
+    **repeated**
+    Lists each ``Party`` that endorsing the candidates associated with the selection. The number of parties is unbounded in cases where multiple parties endorse a single candidate/ticket.
```

Comment:
Should there be some notion here of which party a candidate actually belongs to?